waypoints_station_1: # in degree [base, shoulder, elbow, wrist1, wrist2, wrist3]
  home_pose: [0, -90, 0, -90, 0, 0]
  pre_grasp: [-171.6, -134.6, 120.3, -167.5, -86.5, 38]
  grasp: [-174, -116, 113, -177.4, -78.8, 38.1]
  post_grasp: [-174, -98.1, 50.8, -133.1, -79.1, 38.2]
  pre_set: [-186.9, -112.1, -79.9, 10.8, 96.4, 33.6]
  set: [-186.9, -198.8, -50, 67.6, 96.8, 33.8]
  set_down: [-182.9, 160.7, -121.2, 44.5, 87.8, 50.8]
  post_set: [-192.5, -196, -84.4, 99.2, 102.5, 34]

waypoints_station_2: # in degree [base, shoulder, elbow, wrist1, wrist2, wrist3]
  home_pose: [0, -90, 0, -90, 0, 0]
  pre_grasp: [-171.6, -134.6, 120.3, -167.5, -86.5, 38]
  grasp: [-179.9, -104, 101.7, -178.2, -78.6, 39.2]
  post_grasp: [-181.6, -88.7, 41.2, -135.9, -77.7, 40]
  pre_set: [-161.5, -112.1, -80, 10.8, 96.4, 33.6]
  set: [-158.8, -195.2, -60.6, 77, 94, 33.9]
  set_down: [-150.1, -145.3, -130.3, 27.5, 83.2, 55.3]
  post_set: [-153.7, -175.4, -98.1, 95.7, 89.1, 33.8]

waypoints_station_3: # in degree [base, shoulder, elbow, wrist1, wrist2, wrist3]
  home_pose: [0, -90, 0, -90, 0, 0]
  pre_grasp: [-177.3, -131.2, 117, -172.2, -100.2, 40.4]
  grasp: [-190.3, -106.3, 104.7, -178.5, -89.7, 41.4]
  post_grasp: [-189.5, -100, 73.1, -152, -91.3, 40.3]
  pre_set: [-219.1, -112.1, -80, 10.8, 96.4, 33.6]
  set: [-214.3, -194.4, -63.1, 74.8, 97.7, 40.6]
  set_down: [-212.1, -159, -119.8, 42, 84.1, 40.8]
  post_set: [-214.2, -147, -102.1, 66.4, 97.7, 40.6]


backup_pos: [-180, -90, 0, -90, 0, 0]
linear_tcp_speed: 10
linear_tcp_acceleration: 5
joint_speed: 10
joint_acceleration: 5
# Hand/Gripper
publisher_name: "/hand/SModelRobotOutput"
subscriber_name: "/hand/SModelRobotInput"
hand_server_name: "/hand/hand_action_server"
hand_mode: "basic"